content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Agent Configuration
+weight: 45
+spelling: cSpell:ignore distro mkdir uninstrumented virtualenv
+---
+
+Automatic instrumentation with Python uses a Python agent binary that can be attached to any Python application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks.
```

Comment:
Since this page is about the configuration of the agent, I would say this either needs to be merged into the _index.md or removed. Then have an introduction to "configuration"